components/Image/my-map-test.tsx

```diff
@@ -225,39 +240,87 @@ export default function MyMapTest({ source }: { source: string }) {
     setIsDragging(false)
     setIsFirstPress(true)
     setAccel(1)
+    setTouchDist(0)
     console.log('Mouse LEAVE Event function')
```

Comment:
Is this needed? You have a bunch of console.logs everywhere. I wonder if you need all of them.

Reply:
To be deleted